mllib/src/main/scala/org/apache/spark/mllib/linalg/BLAS.scala

```diff
@@ -228,6 +228,29 @@ private[spark] object BLAS extends Serializable with Logging {
     }
     _nativeBLAS
   }
+ 
+  /**
+   * A := alpha * x * x^T + A
+   */
+  def dsyr(alpha: Double, x: DenseVector, A: DenseMatrix) {
```

Comment:
Can you please rename this to ```syr``` (no "d") since the "d" refers to double-precision (and the type is specified by the given types anyways)?